`actor control list` cmd panics when owner is msig

### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
lotus version
Daemon:  1.17.2-dev+butterflynet+git.818153a1e.dirty+api1.5.0
Local: lotus version 1.17.2-dev+butterflynet+git.818153a1e.dirty
```

Dirty-version because I have manually set the butterfly-bootstraps and genesis.car.

### Describe the Bug

The `lotus-miner actor control list` command panics when a multisig-owner address is set. Earlier the `lotus-miner actor control list` command would produce the `getting account key: failed to get account actor state for [SP-id] unknown actor code [baf....]` error if the owner was a multisig.

This should have been fixed with https://github.com/filecoin-project/lotus/pull/9072. But it now produces a panic.

### Logging Information

```text
lotus-miner actor control list
panic: runtime error: slice bounds out of range [:9] with length 6

goroutine 1 [running]:
main.glob..func5.1({0x3d71f10, 0x5}, {{0xc000d865c3?, 0x0?}})
        /root/lotus/cmd/lotus-miner/actor.go:539 +0xdc6
main.glob..func5(0xc000277000)
        /root/lotus/cmd/lotus-miner/actor.go:581 +0xd4a
github.com/urfave/cli/v2.(*Command).Run(0x5e38ea0, 0xc000276e80)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/command.go:169 +0x631
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc0006829c0, 0xc000276dc0)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/app.go:490 +0xd0a
github.com/urfave/cli/v2.(*Command).startApp(0x5e3ae20, 0xc000276dc0)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/command.go:287 +0x77b
github.com/urfave/cli/v2.(*Command).Run(0x38d4ba0?, 0xc000454a50?)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/command.go:95 +0xba
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc000682820, 0xc000276c00)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/app.go:490 +0xd0a
github.com/urfave/cli/v2.(*Command).startApp(0x5e3e8a0, 0xc000276c00)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/command.go:287 +0x77b
github.com/urfave/cli/v2.(*Command).Run(0xc000276c00?, 0xc000120700?)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/command.go:95 +0xba
github.com/urfave/cli/v2.(*App).RunContext(0xc0006824e0, {0x44ce780?, 0xc000052d00}, {0xc00004e080, 0x4, 0x4})
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/app.go:341 +0xbc8
github.com/urfave/cli/v2.(*App).Run(...)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.8.1/app.go:247
github.com/filecoin-project/lotus/cli.RunApp(0x39a6760?)
        /root/lotus/cli/helper.go:35 +0x4e
main.main()
        /root/lotus/cmd/lotus-miner/main.go:168 +0x10e5
```

### Repo Steps

1. Be on the master-branch - Set msig as owner-address
2. Run `lotus-miner actor control list`
3. Get panic.

--- a/cmd/lotus-miner/actor.go
+++ b/cmd/lotus-miner/actor.go
@@ -513,7 +513,7 @@ var actorControlList = &cli.Command{
 			}
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:9] + "..."
+				kstr = kstr[:6] + "..."
 			}
 
 			bstr := types.FIL(b).String()

--- a/cmd/lotus-miner/actor.go
+++ b/cmd/lotus-miner/actor.go
@@ -513,7 +513,9 @@ var actorControlList = &cli.Command{
 			}
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:6] + "..."
+				if len(kstr) > 9 {
+					kstr = kstr[:6] + "..."
+				}
 			}
 
 			bstr := types.FIL(b).String()

--- a/cmd/lotus-shed/actor.go
+++ b/cmd/lotus-shed/actor.go
@@ -367,7 +367,7 @@ var actorControlList = &cli.Command{
 
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:9] + "..."
+				kstr = kstr[:6] + "..."
 			}
 
 			bstr := types.FIL(b).String()

--- a/cmd/lotus-shed/actor.go
+++ b/cmd/lotus-shed/actor.go
@@ -367,7 +367,9 @@ var actorControlList = &cli.Command{
 
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:6] + "..."
+				if len(kstr) > 9 {
+					kstr = kstr[:6] + "..."
+				}
 			}
 
 			bstr := types.FIL(b).String()

